robottelo/host_info.py

```diff
@@ -0,0 +1,110 @@
+"""Module that gather several informations about host"""
+import logging
+
+import re
+from cachetools import lru_cache
+
+from robottelo import ssh
+LOGGER = logging.getLogger(__name__)
+
+
+@lru_cache(maxsize=1)
+def get_host_os_version():
+    """Fetchs host's OS version through SSH
+    :return: str with version
+    """
+    cmd = ssh.command('cat /etc/redhat-release')
+    if cmd.stdout:
+        version_description = cmd.stdout[0]
+        version_re = (
+            r'Red Hat Enterprise Linux Server release (?P<version>\d(\.\d)*)'
+        )
+        result = re.search(version_re, version_description)
+        if result:
+            host_os_version = 'RHEL{}'.format(result.group('version'))
+            LOGGER.debug('Host version: {}'.format(host_os_version))
+            return host_os_version
+
+    LOGGER.warning('Host version not available: {!r}'.format(cmd))
+    return 'Not Available'
+
+
+_SAT_6_2_VERSION_COMMAND = (
```

Comment:
@rochacbruno there's no 'satellite' at all in upstream, we only have a katello version. The whole concept of 'satellite' version is wrong here. Not sure what is this used for, but we have a different branch for Sat6.2 and Sat6.1, so I think the code merged to master should aim for 6.2 only.